ocfweb/docs/docs/services/account/content-removal.md

```diff
@@ -0,0 +1,46 @@
+[[!meta title="Content Removal Policy"]]
+
+
+## Introduction
+
+The OCF maintains a strict policy of not removing or modifying content
```

Comment:
I agree "content" is a bad word, but "information" suggests that this document only covers textual data and not media such as videos and images.